stmt/stmt.go

```diff
@@ -22,6 +22,8 @@ type Var struct {
 	Name string
 	// Len is the length of the replaced variable.
 	Len int
+	// Exist indicates whether the variable has been defined.
+	Exist bool
```

Comment:
`Exist` is a bit generic, how about `IsDefined`?
```suggestion
	IsDefined bool
```

Reply:
`Defined` ...